tests/test_backends.py

```diff
@@ -64,6 +84,15 @@ def sftp_backend_created():
     finally:
         be.destroy()
 
+@pytest.fixture(scope="function")
+def rclone_backend_created():
```

Comment:
this needs to be added in `pytest_generate_tests` or it won't be invoked for the tests within this test module.

Reply:
Not sure if github has rclone already installed, but in case it has not this could be add to our `.github/workflows/ci.yml`:

```
    - name: Install Linux packages
      run: |
        sudo apt-get update
        sudo apt-get install -y rclone
```

Reply:
pytest_generate_tests ran a lot more tests after that change and I found a lot of things to fix!

I've put the rclone install into the next revision.

The rclone version in ubuntu 22.04 is too old but the one in 24.04 is fine. I put a version check in the backend too so it fails gracefully and the tests should skip on ubuntus which have an old version.